Tests/WSSRV004.ps1

```diff
@@ -0,0 +1,84 @@
+#requires -Modules ExchangeAnalyzer
+
+#This function checks the pagefile min and max sizes
+Function Run-WSSRV004()
+{
+
+   [CmdletBinding()]
+    param()
+
+    $TestID = "WSSRV004"
+    Write-Verbose "----- Starting test $TestID"
+
+    $PassedList = @()
+    $FailedList = @()
+    $WarningList = @()
+    $InfoList = @()
+    $ErrorList = @()
+
+    foreach ($server in $exchangeservers) {
+        $name = $server.name
+
+        # Get RAM of remote server
+        $RAMinMB = (Get-WmiObject -Class win32_physicalmemory -computer $name | measure-object -property capacity -sum).sum/1048576
+        write-verbose "The $name server has $RAMinMB MB of RAM installed"
+
+        # Get IP Address of remote server
+        $ipV4 = Test-Connection -ComputerName $name -Count 1  | Select IPV4Address
+        $ip = $ipV4.IPV4Address
+
+        #Pagefile check
+        $managed = Get-WmiObject -ComputerName $server -Class Win32_ComputerSystem | % {$_.AutomaticManagedPagefile}
+        $RAMIdeal = $RAMinMB+10
+        $CurrentPageFile = (Get-WmiObject -computer $server Win32_PageFileUsage).allocatedbasesize
```

Comment:
I think it's a mistake to assume that you can only get a single IP address as a return. There are a couple of problems with that:

[1] dedicated replication network
[2] dedicated backup network
[3] network holding APIPA addresses because the interfaces are not disabled.